indico/modules/events/editing/blueprint.py

```diff
@@ -26,6 +26,8 @@
 
 # Event-level APIs
 _bp.add_url_rule('/editing/api/file-types', 'api_file_types', backend.RHEditingFileTypes)
+_bp.add_url_rule('/editing/api/file-types/<int:file_type_id>',
+                 'api_edit_file_type', backend.RHEditFileType, methods=('PATCH', 'DELETE'))
```

Comment:
the endpoint fits nicely in the previous line - we use up to 120 chars per line, so no need to wrap that early